### PR TITLE
modify regular expression to match windows disk

### PR DESCRIPTION
**Pull request by XueqiangWei**
Signed-off-by: Xueqiang Wei <xuwei@redhat.com>
ID: 1644589

### CODE REVIEW

**Comment by XueqiangWei** [comment]
(1/8) Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.one_disk_repeat: PASS (235.28 s)
 (2/8) Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.max_disk: PASS (767.59 s)
 (3/8) Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.all_drive_format_types: PASS (221.75 s)
 (4/8) Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.block: PASS (284.16 s)
 (5/8) Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic: PASS (255.95 s)
 (6/8) Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_lun: PASS (1421.74 s)
 (7/8) Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_scsiid_lun: PASS (3572.54 s)
 (8/8) Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_bus_scsiid_lun: PASS (978.06 s)

 (1/8) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.one_disk_repeat: PASS (281.18 s)
 (2/8) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.max_disk: PASS (1094.40 s)
 (3/8) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.all_drive_format_types: PASS (219.42 s)
 (4/8) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.block: PASS (333.09 s)
 (5/8) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic: PASS (291.80 s)
 (6/8) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_lun: PASS (2757.34 s)
 (7/8) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_scsiid_lun: PASS (7365.46 s)
 (8/8) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_bus_scsiid_lun: PASS (1763.52 s)

File: virttest/utils_disk.py [hunk 1]
```diff
@@ -309,7 +309,7 @@ def update_windows_disk_attributes(session, dids, timeout=120):
             if status != 0:
                 logging.error("Can not clear readonly bit: %s" % output)
                 return False
-        pattern = "DISK %s.*Offline" % did
+        pattern = "Status.*Offline"
         if re.search(pattern, details, re.I | re.M):
```
Comment (ybduan) [inline virttest/utils_disk.py]: I think we can discard 'pattern'.
```python
    if re.search("Status.*Offline", details, re.I | re.M):
```

Comment (XueqiangWei) [reply virttest/utils_disk.py]: Agree with you, I will update. Thank you.

**Comment by ybduan** [comment]
Host_RHEL.m8.u0.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.pci_bridge.block_stress.one_bridge: PASS (1507.37 s)

Host_RHEL.m7.u6.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.pci_bridge.block_stress.one_bridge: PASS (1401.67 s)

**Comment by ybduan** [comment]
LGTM
Acked-by: Yanbin Duan yduan@redhat.com

**Comment by YongxueHong** [comment]
Acked-by: Yongxue Hong <yhong@redhat.com>

**Pull request merged**